daemon/graphdriver/windows/windows.go

```diff
@@ -47,6 +48,10 @@ type Driver struct {
 	active map[string]int
 }
 
+func ensureDriverInterface(d *Driver) graphdriver.DiffGetterDriver {
```

Comment:
You're right, this isn't strictly necessary. I just put this in as a trick to validate at compile time that Driver implements the full DiffGetterDriver interface. That way if someone changes something in the interface without updating this code, we won't accidentally fall back to Driver or ProtoDriver.

Should I take it out?